saskatoon/member/autocomplete.py

```diff
@@ -106,3 +106,21 @@ def get_queryset(self):
             qs = qs.filter(q0 | q1 | q2)
 
         return qs.distinct()
+
+
+class EquipmentPointAutocomplete(autocomplete.Select2QuerySetView):
+    """Organizations that are Equipment Points"""
```

Comment:
let's clarify we return an `Actor` queryset (and not `Organization`) as it is being used for the `owner` field